### ISSUE TITLE
VALIDATORS_REWARD_ACCOUNTS can be omitted for native-to-erc mode

### ISSUE THREAD

**Issue by akolotov** [poanetwork/poa-bridge-contracts#271]
Currently `VALIDATORS_REWARD_ACCOUNTS` should be configured for `native-to-erc` mode even if both `HOME_REWARDABLE` and `FOREIGN_REWARDABLE` are `false`. But this requirement is not correct. There is no need to configure accounts to get reward for bridge operations if fee manager is not used.

https://github.com/poanetwork/poa-bridge-contracts/blob/048f51e0e8e7ce4213b727c0982aadab6c994db9/deploy/src/native_to_erc/home.js#L27

**Comment by akolotov** [comment]
If the parameter is not specified in the `.env` file the following error appear:
```
Bridge mode: NATIVE_TO_ERC
Error: TypeError: Cannot read property 'split' of undefined
   at Object.<anonymous> (/home/ubuntu/poa-bridge-contracts/deploy/src/native_to_erc/home.js:27:67)
   at Module._compile (module.js:653:30)
   at Object.Module._extensions..js (module.js:664:10)
   at Module.load (module.js:566:32)
   at tryModuleLoad (module.js:506:12)
   at Function.Module._load (module.js:498:3)
   at Module.require (module.js:597:17)
   at require (internal/module.js:11:18)
   at deployNativeToErc (/home/ubuntu/poa-bridge-contracts/deploy/deploy.js:11:22)
   at main (/home/ubuntu/poa-bridge-contracts/deploy/deploy.js:120:13)
   at Object.<anonymous> (/home/ubuntu/poa-bridge-contracts/deploy/deploy.js:134:1)
   at Module._compile (module.js:653:30)
   at Object.Module._extensions..js (module.js:664:10)
   at Module.load (module.js:566:32)
   at tryModuleLoad (module.js:506:12)
   at Function.Module._load (module.js:498:3)
   at Function.Module.runMain (module.js:694:10)
   at startup (bootstrap_node.js:204:16)
   at bootstrap_node.js:625:3
```

**Comment by akolotov** [comment]
The issue is being closed since it was addressed in the release [`3.0.0`](https://github.com/poanetwork/poa-bridge-contracts/releases/tag/3.0.0)
